internal/storage/postsql/read.go

```diff
@@ -1053,6 +1053,20 @@ func (r readSession) ListInstancesArchived(filter dbmodel.InstanceFilter) ([]dbm
 	return instancesArchived, len(instancesArchived), totalCount, nil
 }
 
+func (r readSession) GetBindingsStatistics() (dbmodel.BindingStats, error) {
+	dto := dbmodel.BindingStats{}
+	statement := r.session.Select("max(extract(epoch from AGE(now(), expires_at))) as max_expiration_time_in_seconds").From(BindingsTableName)
```

Comment:
This is maximum expiration in seconds observed in this point of time. Am I correct?
Not greater than currently set maximum expiration timespan.